pkg/reconciler/source/reconciler.go

```diff
@@ -31,7 +31,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/tools/cache"
 	"knative.dev/pkg/controller"
-	"knative.dev/pkg/logging"
+	logging "knative.dev/pkg/logging"
```

Comment:
nit: Is this necessary?

Reply:
This is an auto formatting side effect. I'll investigate why it's happening. I thought it was the results of goimports

Reply:
No big deal. Just thought it was strange.

Reply:
I'll have to look for these in the future. My guess was this is the result of importing packages with the same name and then unimporting them.